_posts/2018-09-08-terminal-tips.md

```diff
@@ -28,6 +28,13 @@ This command creates a file with the date of the day. %d is for the day, %m is f
 ---
 
 *Contributed by: Bhargav SNV*
+### Stop/Start Displaying Terminal Input  
+    export PS1="MyPrompt >"  
+This let's you edit your prompt text to anything you'd like! Example:
+    
+    root@ubuntu:~# export PS1="MyPrompt >"  
+    Myprompt >  
+    
 ### Stop/Start Displaying Terminal Input
```

Comment:
It's helpful for when you need to timestamp files. Rather than look at the date, copy and then manually type it in. This is far better. Also, automation xD

Reply:
Well it helps to customize your prompt. Also if you have remote sessions like through ssh, it gets hard to differentiate between which terminal is yours. This helps then too...